### PR TITLE
Need tags styling

### LINKED ISSUE #544: Add colour, change style of needs tags
screen shot 2018-07-11 at 2 12 05 pm

### PR DESCRIPTION
**Pull request by smcmurtry**
closes #544

### CODE REVIEW

**Comment by sastels** [comment]
Is the colour of the tag supposed to be the same as the button?
d80ca4ac-6948-4864-85ed-394a0e5c49b7

File: components/need_tag.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  }
+});
+
+export class NeedTag extends Component {
+  render() {
+    const { t, need, classes } = this.props;
+    return (
+      <div
+        className={classes.needsTag}
+        style={{ backgroundColor: need.colour }}
```
Comment (maxneuvians) [inline components/need_tag.js:21]: Very cool! 👍

**Comment by maxneuvians** [comment]
It is the right colour if you are in Option B 

https://vac-poc-staging-pr-547.herokuapp.com/benefits-directory?lng=en&selectedNeeds=recEahND3kwnUPTHx&option=B

**Comment by smcmurtry** [comment]
@sastels should be fixed now

**Review by sastels — APPROVED** [review]
👍

**Pull request merged**